src/blocks/EmbedBlock/embeds/Default.client.ts

```diff
@@ -1,13 +1,13 @@
-const enhanceIntersectedEmbeds = (entries: IntersectionObserverEntry[]) => {
+const enhanceIntersectedDefaultEmbeds = (entries: IntersectionObserverEntry[]) => {
```

Comment:
revert these changes? file is scoped even though editor things there's a conflict with other `.client.ts` files.